DNS/DNS_Cache.c

```diff
@@ -123,10 +123,12 @@
                                     uint32_t * pulIP,
```

Comment:
In the @brief, we need to add the return value description. Also, why didn't doxygen flag this? This file might not be getting included in the doxygen config.